server/models/partyModel.js

```diff
@@ -14,6 +14,10 @@ class PartyModel {
     parties.push(newParty);
     return newParty;
   }
```

Comment:
@frankly034 Nice work! since this file is holding a `class` I will suggest you change the name of the file from `camelCase` to `pascal`

Reply:
Ok, I'll modify all of them. Thanks

Reply:
Thanks for the observation. Modified, committed and pushed.